### ISSUE TITLE
 Issue Report - Error in Step 3 of Optimization Process

### ISSUE THREAD

**Issue by CeliaJuarillo** [UCSF-DSCOLAB/cyclone#15]
I am encountering an error in Step 3 of the optimization process while using your package. The error message is as follows:

##########################################################
Starting step#: 3
[ 2024-02-07 10:15:45 ] Starting the optimization of clustering parameters using following parameters:  
Parameters:
{
	 seed = 123 
	 isWindows = FALSE 
	 data_dir = /home/cjuarez/Escritorio/spectral/files_fcs 
	 out_dir = /home/cjuarez/Escritorio/spectral/intento2/outputs 
	 rerun_from_step = -1 
	 arcsinh_cofactor = 5 
	 subsample = FALSE 
	 subsample_n = 4000 
	 n_neighbors = 15 
	 min_dist = 0.1 
	 spread = 0.1 
	 learning_rate = 0.5 
	 init = random 
	 clustering_method = flowsom 
	 clara_params = 20 
 	 clara_params = euclidean 
 	 clara_params = 50 
	 flowsom_params = /home/cjuarez/Escritorio/spectral/intento2/grid_sizes.csv 
 	 flowsom_params = 6 
 	 flowsom_params = 6 
 	 flowsom_params = 3 
 	 flowsom_params = FALSE 
	 nthreads = 10 
	 scaffold_dir = /home/cjuarez/Escritorio/spectral/intento2/outputs/scaffold 
	 exclude_controls = FALSE 
}
[ 2024-02-07 10:15:45 ] Using the following markers for optimization:  
[1] ""
[ 2024-02-07 10:15:45 ] Optimization Begins: 
[ 2024-02-07 10:15:45 ] Preparing the flowFrame for input to FlowSOM:  
Error: Argument 'exprs' must be numeric matrix with colnames attribute set
Ejecución interrumpida

I have reviewed the parameters and files, but I couldn't identify any issues. Your assistance in resolving this matter would be greatly appreciated. If additional information is required, please let me know. Thank you!

**Comment by ravipatel4** [comment]
@CeliaJuarillo, were you able to resolve the issue? I am unable to see any description of your issue.

**Comment by CeliaJuarillo** [comment]
> @CeliaJuarillo, were you able to resolve the issue? I am unable to see any description of your issue.

Sorry for some reason it got deleted, I have written again the issue for you to see
